File: java/program54.java
import java.util.*;

class CharacterX
{
    public char ch;

    CharacterX(char c)
    {
        this.ch = c;
    }

    public boolean CheckChar()
    {
        if((ch >= 'A') && (ch <= 'Z') || (ch >= 'a') && (ch <= 'z'))
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

class program54
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter character");
        char ch = sobj.next().charAt(0);

        CharacterX obj = new CharacterX(ch);
        boolean bRet = obj.CheckChar();

        if(bRet == true)
        {
            System.out.println("It is Character");
        }
        else
        {
            System.out.println("It is not character");
        }
    }
}

File: cpp/program54.cpp
// program54.c

#include<iostream>
using namespace std;

bool CheckLetter(char cValue)
{
    if((cValue >= 'A') && (cValue <= 'Z'))
    {
        return true;
    }
    else
    {
        return false;
    }
}

int main()
{
    char ch = '\0';
    bool bRet = false;

    cout<<"Enter character \n";
    cin>>ch;

    bRet = CheckLetter(ch);
    if(bRet == true)
    {
        cout<<ch<<" is a capitial letter :\n";
    }
    else
    {
        cout << ch << " is  NOT capitial letter :\n";
    }
}